src/components/Account/connectors/networks.ts

```diff
@@ -0,0 +1,110 @@
+import type { AddEthereumChainParameter } from "@web3-react/types";
```

Comment:
Yes it's used in TemporaryConnectAccount.tsx line 91 in function getAddChainParameters, also just a side note don't worry too much about this code because we will delete it once we have the authentication up and  running